tests/test_flow/conftest.py

```diff
@@ -258,20 +258,32 @@ def __init__(self, caplog):
         self._caplog = caplog
 
     def expect_all(self, *expected_messages):
-        actual_messages = self._pop_messages()
+        actual_messages = self._pop_messages(include_exception=False)
         assert set(actual_messages) == set(expected_messages)
         self._caplog.clear()
 
-    def expect_regex(self, *expected_patterns):
-        actual_messages = self._pop_messages()
+    def expect_regex(self, *expected_patterns, include_exception=False):
```

Comment:
Is there any reason not to always include exceptions?

Reply:
Good point. Changed to always include exceptions.